http/src/main/resources/jupyter/init-vm.sh

```diff
@@ -1,170 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
👍 . I guess this logic now just lives here: https://github.com/DataBiosphere/terra-docker/blob/master/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh